src/core/lib/iomgr/tcp_server_utils_posix_common.cc

```diff
@@ -72,8 +72,9 @@ static void init_max_accept_queue_size(void) {
   s_max_accept_queue_size = n;
 
   if (s_max_accept_queue_size < MIN_SAFE_ACCEPT_QUEUE_SIZE) {
-    LOG(INFO) << "Suspiciously small accept queue (" << s_max_accept_queue_size
-              << ") will probably lead to connection drops";
+    LOG(WARNING) << "Suspiciously small accept queue ("
+                 << s_max_accept_queue_size
+                 << ") will probably lead to connection drops";
   }
```

Comment:
Same as above. Changing INFO to WARNING feels out of place as a part of a log noise reduction PR.

Reply:
reverting